src/api/courseOperations.tsx

```diff
@@ -0,0 +1,41 @@
+import { Course } from 'api';
+import { AddCourseData } from 'components/AddCourseModal';
+
+const handleErrors = (response: Response) => {
+	if (!response.ok) {
+		throw Error('fetch unsuccessful');
+	}
+	return response;
+};
```

Comment:
We should add some utils functions to the `api` stuff if we are going to use the same code for all of the API calls. 

I believe we should capture some work to use the API response for something in the future. Maybe display a toast pop up of what happened, based on what the API responds.